src/util.cpp

```diff
@@ -801,11 +801,9 @@ int GetNumCores()
 
 std::string CopyrightHolders(const std::string& strPrefix)
 {
-    std::string strCopyrightHolders = strPrefix + _(COPYRIGHT_HOLDERS);
-    if (strCopyrightHolders.find("%s") != strCopyrightHolders.npos) {
-        strCopyrightHolders = strprintf(strCopyrightHolders, _(COPYRIGHT_HOLDERS_SUBSTITUTION));
-    }
-    if (strCopyrightHolders.find("Bitcoin Core developers") == strCopyrightHolders.npos) {
+    std::string strCopyrightHolders = strPrefix + strprintf(_(COPYRIGHT_HOLDERS), _(COPYRIGHT_HOLDERS_SUBSTITUTION));
+    if (std::string("Bitcoin Core") != COPYRIGHT_HOLDERS_SUBSTITUTION) {
```

Comment:
Bitcoin Core developers could very well  be mentioned in COPYRIGHT_HOLDERS rather than COPYRIGHT_HOLDERS_SUBSTITUTION. For example:

```C++
 define(_COPYRIGHT_HOLDERS,[The %s and Bitcoin Core developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[Altcoin Core])
```